### DISCUSSION TITLE
Upc Renault Talisman

### DISCUSSION THREAD

**Discussion by JohnnyX99** [cedricp/ddt4all#1071] [Q&A]
Hello
I buy an used Upc for my Talisman I try from Renolink to change vin was unsuccessful and now I can't access the modul I receive an error incorrect data  string or something like that .i use the option from change vin number in Renolink.Its possible to make something now ?

**Comment by JohnnyX99** [comment]
![20240407_202301_copy_1900x1425](https://github.com/cedricp/ddt4all/assets/162407382/92905406-2674-425f-b581-6d4a84956af4)
![20240407_202250_copy_1413x1884](https://github.com/cedricp/ddt4all/assets/162407382/bd37e2ac-e337-48c7-a8f3-598ff9b94d59)
Pictura with Usm Upc

**Comment by matzel82** [comment]
You have try to change the vin. This is not possible. To use Renolink (I have it too, but only not the version from the original developer 🫤) can make problems. Better you buy a new original UPC and your Garage install the latest SW and config it. So it would be work without problems.

**Comment by JohnnyX99** [comment]
I have the original version of Renolink 2.05 with Obdlink Ex original

**Comment by KarelSvo** [comment]
Why do you show the control unit and not the error message? Tried you connected with Pyren?

**Reply by JohnnyX99** [reply]
I can't connect anymore I think a reflash will solve this
